recipes/thucyd/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "thucyd" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "e3c0c736746b0ac26b93fa8d2fcd2d90394adb080d60a84144aabf9f83325381"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy >=1.14
+    - pip
+    - python >=3.5
+  run:
+    - numpy >=1.14
+    - python >=3.5
+
+test:
+  imports:
+    - thucyd
+
+about:
+  home: "https://gitlab.com/thucyd-dev/thucyd"
+  license: "Apache 2.0"
```

Using the SPDX names of the licences is useful for consistency:
```suggestion
  license: Apache-2.0
```

---

Sure, I'll change this.

---

Let me know about the comment above and then I'll push the resolution changes.

---

committed and built.